### DISCUSSION TITLE
Development will be slowing down

### DISCUSSION THREAD

**Discussion by jordanbaird** [jordanbaird/Ice#424] [Announcements]
When I first created Ice, I decided to release it for free and make it open source. I firmly believe it is this decision that has enabled Ice's userbase to grow at the incredible rate it has. While I am immensly grateful for the support I've received over the last several months, unfortunately, kind words don't pay the bills. The reality is, most people who use the app do so without providing a financial contribution. While I want to make it clear that nobody is obligated to give me money, as a human being, I need to be able to support myself financially. As such, I have accepted a full time position with a company, and will be developing their macOS app. While I will try to continue to provide support and updates for Ice, I want to make it clear that most of my time in the forseeable future will be dedicated to my job.

I want to give a heartfelt thanks to anyone who has donated or purchased the app. Without your support, Ice would not be where it is today. I also want to make it clear that I am not abandoning this app. Development is merely slowing down. Thank you for your understanding.

Jordan Baird,
Creator of Ice

**Comment by nbently** [comment]
Sell features in a subscription at a premium — I'd pay for it! I had no problem paying for bartender.

**Reply by maxoliinyk** [reply]
What's the point of this app then if we'll have subscription like in tens of other menubar management apps? The whole reason most people started using this application is the FOSS model

**Reply by AngryDingo** [reply]
> What's the point of this app then if we'll have subscription like in tens of other menubar management apps? The whole reason most people started using this application is the FOSS model

That's not why I started using Ice and I'm sure I'm not the only one.

**Reply by bst1n** [reply]
+1

**Comment by TheWeirdSapien** [comment]
Thank you, Ice has been an excellent alternative to bartender, Honestly very happy with where it is, so thank you for your work! Managing my menubar without it seems barbaric. Good luck at your new job!

**Comment by pketh** [comment]
I've been really enjoying ice as an alternative to bartender too. tbh before this post, I didn't even realize you could pay for it. Just left a payment at https://buymeacoffee.com/jordanbaird

**Reply by Imolos** [reply]
Hey thanks for the link I started by sponsoring trough Github but that model doesn't sits with me. 
This looks better

**Comment by getninjaN** [comment]
Good luck on your new job!
I'll keep sponsoring.

**Comment by mathewmitchell** [comment]
Similar to some others, I didn't even realize one could contribute $$$. It's very well hidden ... I'd suggest making it MUCH more apparent. I don't think it's because people didn't want to donate ... aside from a small number. More it's an issue that it's not even clear how to pay ... needs to be MUCH more obvious.

At any rate, just bought you 3 coffees!

**Comment by WeiChongDevelops** [comment]
Thanks for your hard work and good luck at your job Jordan! They're lucky to have you.

**Comment by ykrem** [comment]
Canonical macos app. Same as others, missed the donation links. Thanks for ice!

**Comment by Sly777** [comment]
Good luck at your new job! I also heard first time there was a link for buy or support. I was even thinking why ice doesn't ask for anything 😄

**Comment by sadfish161** [comment]
Congratulations! I hope you won’t drop Ice (that would make Sadfish more sad than I am already) but also understand focusing on what pays bills.

I wanted to support development via Gumroad but they don’t accept PayPal or prepaid cards. Why not place some features of Ice behind a paywall? Many of us were already accustomed to spending as much as $100 for an alternative yet yours has been the best + free(?!). I’d gladly pay for a license if you implemented a service that accepted other forms of payment.

**Comment by berters** [comment]
@jordanbaird I'd be happy for you to make this paid. I had 2 big issues with Bartender; scrolling or clicking the menu bar to reveal hidden items would open various menus from icons. e.g Krisp or Akiflow would have their menus pop open. This issue has existed for the longest time without a fix. Then a recent bug caused the Apple Menu to open on its own periodically.

I have just tried Ice and face neither of these issues, it just works as expected.

**Comment by iamjoeker** [comment]
Like others, I missed the links to pay for it as well. I absolutely love this app! I find it very effective in reducing distractions without hiding everything. Sent you some coffee. Best of luck in your new position!

**Comment by hypn-a** [comment]
Pretty much agreeing with the others commenting before me. Only discovered Ice two weeks ago and it has been my menu manager of choice. Missed the donation option all together but will now contribute. Like many others I have no problem if the app. does requests a monthly admin fee

**Reply by rebdeg** [reply]
I fully agree. I also don't have any problems paying a monthly fee. I sincerely hope that @jordanbaird will be able to pick up coding ICE again soon.

**Reply by qwertyus124** [reply]
Seconded!

**Comment by ChrisGVE** [comment]
You could check whether you could distribute Ice via Setapp, it could give you a stable income. But I dont know the requirement.

Thanks for making this apps!

**Reply by BryanWall** [reply]
This is a great idea

**Comment by alexzadeh** [comment]
Hi @jordanbaird, I love Ice and happy to pay!

**Comment by murtuzaalisurti** [comment]
@jordanbaird  the most useful mac app, featured on my blog https://syntackle.com/blog/the-most-useful-macos-app/
bought you a coffee ❤️

**Comment by BryanWall** [comment]
Add me to the list of people who would have contributed financially if I had known there was a way to do so. I just sent you a coffee and would be happy to contribute regularly if you keep the app maintained and working...

You should consider putting Ice on Setapp. It should provide some low effort ongoing revenue, and exposure to a lot more potential users. Here's a link to the info:
https://setapp.com/developers

**Comment by Cippa** [comment]
Just reading now. A pity that people do not contribute; work has a value and it needs to be recognised. Otherwise, there is no possibility for developers to build good software. This in turn affects us, as consumers, that we are not willing to make a small donation but that we want to have everything for free. In most cases we end up in using free apps that sell all our data...

**Comment by A-0-0-P** [comment]
Thanks for all the hard work so far, Jordan. I just got you 3 coffees. 
It's understandable that you can't do it all for free, we need to earn a living — hopefully more people will send you some $. 
I have to admit that the possibility to donate was also very hidden for me. Maybe an idea to add it to the menu? I'm sure more people will send you some  ☕

<img width="336" height="240" alt="SCR-20251105-kost" src="https://github.com/user-attachments/assets/6d35153d-c677-447d-be0b-7133f1a8c7a4" />
